core/src/main/java/io/questdb/griffin/engine/table/LatestByValuesRecordCursor.java

```diff
@@ -69,5 +77,10 @@ private void prepare() {
         for (int i = 0, n = symbolKeys.size(); i < n; i++) {
             map.put(symbolKeys.get(i), 0);
         }
+        if (deferredSymbolKeys != null) {
```

Comment:
suspicious that `map` is never cleaned and re-evaluated

Reply:
That's a good point. Fixed that in 4cfbdc97ba000ed59637388dccbe6938c83db062

The same problem is present in `FilterOnExcludedValuesRecordCursorFactory`, but this PR doesn't change that class. I'm going to fix it in a separate one.